confdb/ax_check_posix_regcomp.m4

```diff
@@ -35,7 +35,7 @@ int main() {
     regex_t  re;
     if (regcomp( &re, "^.*$", flags  ) != 0)
         return 1;
-    return regcomp( &re, "yes.*|no.*", flags  ); 
+    return regcomp( &re, "yes.*|no.*", flags  );
```

Comment:
This should be part of the previous commit. It doesn't relate to the libslurm addition.

Reply:
@raffenet hopefully this time I got it right